test/source/browser/controllable.ts

```diff
@@ -683,6 +683,10 @@ export class ControllablePage extends ControllableBase {
     return html;
   };
 
+  public sleep = async (seconds: number) => {
+    return await Util.sleep(seconds);
+  };
```

Comment:
Why not use `Util.sleep` directly? Having another method doesn't add value.

Reply:
Got it, but that doesn't justify duplicating functionality. Like this, the code will bloat. It's a small method here and a small method there, and before we know it the codebase is triple the size.

If the method is in the wrong place, it should be moved, not duplicated. It would be technically incorrect to have it on a controllable (because it's not just one page that's sleeping, it's the test itself sleeping). But it would be marginally more convenient.

@rrrooommmaaa maybe you have some thoughts

Reply:
I usually just type `Util.sleep` and `vscode` import `Util`. So adding `sleep` to `Controllable` in my opinion isn't justified

Reply:
Yes, that's it, but when you then comment out it or remove, and it is no longer used, import is not auto-removed (unfortunately), and this causes build error due to linters, so you have to do it manually, and it's quite annoying if you have to do it multiple times in a row, which happens often during debugging. Ok, I will remove, but it is then reasonable to keep a dummy function or at least one call to it that it is not necessary to remove import manually.